src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.ts

```diff
@@ -191,6 +196,8 @@ export class CreateEditNewsComponent extends FormBaseComponent implements OnInit
         })
       )
       .subscribe(() => this.escapeFromCreatePage());
+
+    localStorage.removeItem('newsTags');
```

Comment:
we should use localStorageSerrvice for this

Reply:
done